parser-typechecker/src/Unison/Codebase/FileCodebase/Reserialize.hs

```diff
@@ -70,12 +70,13 @@ syncToDirectory
   -> Branch m
   -> m ()
 syncToDirectory fmtV fmtA srcPath destPath branch = do
-  flip evalStateT mempty $
+  _written <- flip execStateT mempty $
```

Comment:
Why?  Half-logged I guess?

Reply:
Yeah... there was some logging in here originally. Didn't bother with cleanup since we're deleting.